src/helpers/getSliceCartUsdcPrice.ts

```diff
@@ -0,0 +1,42 @@
+import { wagmiConfig } from "./constants";
+import { CartItem } from "./sliceKitTypes";
+import {
+  generateCart,
+  getStoreProducts,
+  getTotalPrices,
+} from '@slicekit/core';
+
+type Props = {
+  slicerId: number;
+  cartParams: CartItem[];
+}
+
+export async function getSliceCartUsdcPrice({ slicerId, cartParams }: Props) {
+  // construct the cart using the slicekit to calculate the total amount
+  const { cartProducts: allProducts } = await getStoreProducts(wagmiConfig, {
+    slicerId,
+  });
+
+  console.log({ cartParams });
+
+  const cart = generateCart({
+    cartParams,
+    allProducts,
+  });
+
+  console.log({ cart });
```

Comment:
Remember to remove the console logs after we're sure implementation is the correct one